content/community/team.md

```diff
@@ -26,6 +26,7 @@ currently contributing to Kokkos Core and Kokkos Kernels below.
 {{< image src="img/CSCS-Logo-258X79.jpg"           class="community-team-image-small">}}
 {{< image src="img/LANL-logo-247x89.jpg"           class="community-team-image-medium">}}
 {{< image src="img/LBNL-logo-316x240.jpg"          class="community-team-image-xsmall">}}
+{{< image src="https://my.com.uliege.be/upload/docs/image/png/2018-05/uliege_logo_compact_cmjn_pos2x.png" class="community-team-image-medium">}}
```

Comment:
<img width="625" height="677" alt="Screenshot 2025-09-30 at 11 33 42 AM" src="https://github.com/user-attachments/assets/1208bec7-f06c-446e-a7cb-61ec12339630" />
You need to authenticate to see the page